clear-containers-image/clear-containers-image.dsc-template

```diff
@@ -0,0 +1,16 @@
+Format: 3.0 (quilt)
+Source: clear-containers-image
+Version: @VERSION@-@RELEASE@
+Section: devel
+Priority: optional
+Maintainer: clearlinux.org team <dev@lists.clearlinux.org>
```

Comment:
This should be changed to Clear Containers and external mailing list not clear linux

Reply:
Maintainer: clear containers team <cc-devel@lists.01.org>